libs/shell/shell.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _SHELL_H_
+#define _SHELL_H_
+
+struct shell_cmd {
+	char *command;
+	void(*action)(int argc, char **argv);
+	void(*help)(void);
+};
+
+#define SHELL_MAX_COMMANDS 10
+
+void shell_init(void);
+void shell_process(void *params);
```

Comment:
Missed that.  Shell process used to be initialized directly, but I rolled it into init.  I'll remove that.